core/dbt/utils.py

```diff
@@ -454,7 +454,7 @@ def __get__(self, obj, objtype):
 
 
 def format_bytes(num_bytes):
-    for unit in ["Bytes", "KB", "MB", "GB", "TB", "PB"]:
+    for unit in ["Bytes", "KiB", "MiB", "GiB", "TiB", "PiB"]:
```

Comment:
I don't think we actually use this method anywhere in `dbt-core`. It's a holdover from when it was added, for the sole benefit of `dbt-bigquery`, and we forgot to remove it during the Great Adapter Split-Apart of 2021.

What do you think about just removing it?

Reply:
> What do you think about just removing it?

Removed!

I did a quick cursory check for both `format_bytes` and `format_rows_number` in the dbt Labs adapters using a GitHub search like [this](https://github.com/dbt-labs/dbt-redshift/search?q=format_rows_number&type=code). DIdn't see any references other than in dbt-bigquery, which we know about.

I didn't check any non-dbt Labs adapters, although our expectation is that it is not being used. If it _did_ happen to be used by some adapter out there, they'd be on their own to resolve it.